services/scan/src/central_scanner_app.ts

```diff
@@ -197,21 +197,21 @@ export async function buildCentralScannerApp({
     response.json({ status: 'ok' });
   });
 
-  app.get<NoParams, Scan.GetCurrentPrecinctConfigResponse>(
+  app.get<NoParams, Scan.GetPrecinctSelectionConfigResponse>(
```

Comment:
Does the central scanner use these endpoints at all? As far as I can tell, no. And I don't see why it would need them since it's not configurable by precinct. They can probably be removed (along with the corresponding functions in frontends/bsd/src/api/config.ts)

Reply:
No, we don't use them, wasn't sure if I should remove them. I will go ahead and do so.

Reply:
Removed.